public/server.py

```diff
@@ -1,17 +1,15 @@
 #!/usr/bin/env python
-import http.server
-import socketserver
-class MyHttpRequestHandler(http.server.SimpleHTTPRequestHandler):
+import SimpleHTTPServer, SocketServer
+class MyHttpRequestHandler(SimpleHTTPServer.SimpleHTTPRequestHandler):
     def do_GET(self):
         if self.path == '/status':
             self.send_response(200)
-        else: 
-            self.path = '/index.html'
-            return http.server.SimpleHTTPRequestHandler.do_GET(self)
+        else:
+            return SimpleHTTPServer.SimpleHTTPRequestHandler.do_GET(self)
```

Comment:
So here, it's calling the inherited function - I think you can do something like `super().do_GET()` 🤔 
Which means you're calling the parent function, and I think the "self" call might be implicit.

Obviously if the code works I'm also quite happy as it is, just curious I suppose :)

Reply:
This is a very cut and pasted solution so any improvements welcome, thanks!

Reply:
I tried this suggestion and it didn't work :(